client/configuration_set_test.go

```diff
@@ -152,3 +155,5 @@ var _ = Describe("Configuration Set", func() {
 		})
 	})
 })
+
+// TODO add more tests...
```

Comment:
 todo?

Reply:
that's leftovers... forgot to remove it...

Reply:
will remove it on the next PR.